### PR TITLE
[docs] v1.13.0 changelog

### PR DESCRIPTION
**Pull request by DIRECTcut**

### CODE REVIEW

**Review by Farfurix — APPROVED** [review]

File: docs/articles/blog/2021-3-03-testcafe-v1-13-0-released.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+TestCafe now enables you to specify a custom path to the [configuration file](../documentation/reference/configuration-file.md).
+
+To set the path, use one of the following options:
+
+* the [*--config-file* CLI option](../documentation/reference/command-line-interface.md#--config-file-path)
+* the [*configFile* option of the *createTestCafe* function](../documentation/reference/testcafe-api/global/createtestcafe.md#options)
+
+### Add Support for Microsoft Edge on Linux ([PR testcafe-browser-tools/#210](https://github.com/DevExpress/testcafe-browser-tools/pull/210) by [@dcsaszar](https://github.com/dcsaszar))
+
+If you follow the [Microsoft Edge Insider Channels for Linux](https://www.microsoftedgeinsider.com/en-us/download?platform=linux-deb) and have Microsoft Edge installed on your Linux machine, you can now launch TestCafe tests in this browser.
```
Comment (VasilyStrelyaev) [inline docs/articles/blog/2021-3-03-testcafe-v1-13-0-released.md:25]: Hm, looks like you should ***follow*** a channel to launch TestCafe tests...

Would it be clearer to say that you can launch tests in a browser version from the Insider channel (if that's what you mean)?

File: docs/articles/blog/2021-3-03-testcafe-v1-13-0-released.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+* Fixed an error where a `documentElement.transform.translate` call moved the TestCafe UI in the browser window ([#5606](https://github.com/DevExpress/testcafe/issues/5606))
+* TestCafe now emits a warning if you pass an unawaited selector to an assertion ([#5554](https://github.com/DevExpress/testcafe/issues/5554))
+* Fixed a crash that sometimes occurred in Chrome v85 and earlier on pages with scripts ([PR testcafe-hammerhead/#2590](https://github.com/DevExpress/testcafe-hammerhead/pull/2590))
+* Overrode the [insertAdjacentElement method](https://developer.mozilla.org/en-US/docs/Web/API/Element/insertAdjacentElement) inside TestCafe ([testcafe-hammerhead/#1321](https://github.com/DevExpress/testcafe-hammerhead/issues/1321))
+* Overrode the internal `Element.prototype.after` method ([testcafe-hammerhead/#1833](https://github.com/DevExpress/testcafe-hammerhead/issues/1833))
+* Overrode the [ParentNode.prepend method](https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/prepend) inside TestCafe ([testcafe-hammerhead/#1005](https://github.com/DevExpress/testcafe-hammerhead/issues/1005))
```
Comment (VasilyStrelyaev) [inline docs/articles/blog/2021-3-03-testcafe-v1-13-0-released.md:80]: Are these three fixes relevant to users?

File: docs/articles/blog/2021-3-03-testcafe-v1-13-0-released.md (hunk 1 above)
Comment (DIRECTcut) [reply docs/articles/blog/2021-3-03-testcafe-v1-13-0-released.md:25]: > Would it be clearer to say that you can launch tests in a browser version from the Insider channel?

Yes, that's right.

Is that better:

```suggestion
If you have the [Insider version of the Microsoft Edge](https://www.microsoftedgeinsider.com/en-us/download?platform=linux-deb) installed on your Linux machine, you can now launch TestCafe tests in this browser.
```

File: docs/articles/blog/2021-3-03-testcafe-v1-13-0-released.md (hunk 2 above)
Comment (DIRECTcut) [reply docs/articles/blog/2021-3-03-testcafe-v1-13-0-released.md:80]: I think we can safely omit those

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,76 @@
...
 
+## v1.13.0 (2021-03-22)
+
+This release adds support for custom paths to the configuration file, support for Microsoft Edge on Linux systems, and multiple bugfixes.
+
+### Enhancements
+
+#### :gear: Specify Custom Path to the TestCafe Configuration File ([PR #6035](https://github.com/DevExpress/testcafe/pull/6035) by [@Chris-Greaves](https://github.com/Chris-Greaves))
+
+TestCafe now enables you to specify a custom path to the [configuration file](../documentation/reference/configuration-file.md).
```
Comment (dirk-pieterse) [inline CHANGELOG.md:11]:
```suggestion
TestCafe now allows you to specify a custom path to the [configuration file](../documentation/reference/configuration-file.md).
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,76 @@
...
+
+If you follow the [Microsoft Edge Insider Channels for Linux](https://www.microsoftedgeinsider.com/en-us/download?platform=linux-deb) and have Microsoft Edge installed on your Linux machine, you can now launch TestCafe tests in this browser.
+
+```sh
+testcafe edge tests/test.js
+```
+
+#### :gear: Deprecated the `t.setPageLoadTimeout` method ([PR #5979](https://github.com/DevExpress/testcafe/pull/5979))
+
+Starting v1.13.0, the [t.setPageLoadTimeout](../documentation/reference/test-api/testcontroller/setpageloadtimeout.md) method is deprecated. To set the page load timeout, use the new [test.timeouts](../documentation/reference/test-api/test/timeouts.md) method.
```
Comment (dirk-pieterse) [inline CHANGELOG.md:28]:
```suggestion
Starting with v1.13.0, the [t.setPageLoadTimeout](../documentation/reference/test-api/testcontroller/setpageloadtimeout.md) method is deprecated. To set the page load timeout, use the new [test.timeouts](../documentation/reference/test-api/test/timeouts.md) method.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,76 @@
...
+test
+    .timeouts({
+        pageLoadTimeout: 2000
+    })
+    ('My test', async t => {
+        //test actions
+    })
+```
+
+You can use `test.timeouts` to set the [pageRequestTimeout](../documentation/reference/configuration-file.md#pagerequesttimeout) and [ajaxRequestTimeout](../documentation/reference/configuration-file.md#ajaxrequesttimeout) as well.
```
Comment (dirk-pieterse) [inline CHANGELOG.md:43]:
```suggestion
You can also use `test.timeouts` to set the [pageRequestTimeout](../documentation/reference/configuration-file.md#pagerequesttimeout) and [ajaxRequestTimeout](../documentation/reference/configuration-file.md#ajaxrequesttimeout).
```

**Review by dirk-pieterse — APPROVED** [review]

File: docs/articles/blog/2021-3-22-testcafe-v1-13-0-released.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+This release adds support for custom paths to the configuration file, support for Microsoft Edge on Linux systems, and multiple bugfixes.
+
+<!--more-->
+
+## Enhancements
+
+### ⚙ Specify Custom Path to the TestCafe Configuration File ([PR #6035](https://github.com/DevExpress/testcafe/pull/6035) by [@Chris-Greaves](https://github.com/Chris-Greaves))
+
+TestCafe now enables you to specify a custom path to the [configuration file](../documentation/reference/configuration-file.md).
```
Comment (DIRECTcut) [inline docs/articles/blog/2021-3-22-testcafe-v1-13-0-released.md:16]:
```suggestion
TestCafe now allows you to specify a custom path to the [configuration file](../documentation/reference/configuration-file.md).
```

File: docs/articles/blog/2021-3-22-testcafe-v1-13-0-released.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+test
+    .timeouts({
+        pageLoadTimeout: 2000
+    })
+    ('My test', async t => {
+        //test actions
+    })
+```
+
+You can use `test.timeouts` to set the [pageRequestTimeout](../documentation/reference/configuration-file.md#pagerequesttimeout) and [ajaxRequestTimeout](../documentation/reference/configuration-file.md#ajaxrequesttimeout) as well.
```
Comment (DIRECTcut) [inline docs/articles/blog/2021-3-22-testcafe-v1-13-0-released.md:48]:
```suggestion
You can also use `test.timeouts` to set the [pageRequestTimeout](../documentation/reference/configuration-file.md#pagerequesttimeout) and [ajaxRequestTimeout](../documentation/reference/configuration-file.md#ajaxrequesttimeout) as well.
```

File: docs/articles/blog/2021-3-22-testcafe-v1-13-0-released.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+
+If you have [Microsoft Edge installed on your Linux machine](https://www.microsoftedgeinsider.com/en-us/download?platform=linux-deb), you can now launch TestCafe tests in this browser.
+
+```sh
+testcafe edge tests/test.js
+```
+
+### ⚙ Deprecated the `t.setPageLoadTimeout` method ([PR #5979](https://github.com/DevExpress/testcafe/pull/5979))
+
+Starting v1.13.0, the [t.setPageLoadTimeout](../documentation/reference/test-api/testcontroller/setpageloadtimeout.md) method is deprecated. To set the page load timeout, use the new [test.timeouts](../documentation/reference/test-api/test/timeouts.md) method.
```
Comment (DIRECTcut) [inline docs/articles/blog/2021-3-22-testcafe-v1-13-0-released.md:33]:
```suggestion
Starting with v1.13.0, the [t.setPageLoadTimeout](../documentation/reference/test-api/testcontroller/setpageloadtimeout.md) method is deprecated. To set the page load timeout, use the new [test.timeouts](../documentation/reference/test-api/test/timeouts.md) method.
```

File: docs/articles/blog/2021-3-22-testcafe-v1-13-0-released.md [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+test
+    .timeouts({
+        pageLoadTimeout: 2000
+    })
+    ('My test', async t => {
+        //test actions
+    })
+```
+
+You can also use `test.timeouts` to set the [pageRequestTimeout](../documentation/reference/configuration-file.md#pagerequesttimeout) and [ajaxRequestTimeout](../documentation/reference/configuration-file.md#ajaxrequesttimeout) as well.
```
Comment (DIRECTcut) [inline docs/articles/blog/2021-3-22-testcafe-v1-13-0-released.md:48]:
```suggestion
You can also use `test.timeouts` to set the [pageRequestTimeout](../documentation/reference/configuration-file.md#pagerequesttimeout) and [ajaxRequestTimeout](../documentation/reference/configuration-file.md#ajaxrequesttimeout).
```
